src/Entity/User.php

```diff
@@ -65,9 +65,12 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
     #[ORM\OneToMany(mappedBy: 'user', targetEntity: Order::class)]
     private Collection $orders;
 
+    #[ORM\OneToMany(mappedBy: 'userId', targetEntity: Comment::class)]
```

Comment:
I think no need to declare relation between user and comment table directly
but ask mentor.

Reply:
done